src/Proto.Actor/LocalContext.cs

```diff
@@ -31,7 +33,7 @@ public class Context : IMessageInvoker, IContext, ISupervisor
         private bool _stopping;
         private HashSet<PID> _watchers;
         private HashSet<PID> _watching;
-        private readonly ILogger logger = Log.CreateLogger<Context>();
+        private readonly ILogger _logger = Log.CreateLogger<Context>();

```

Comment:
The logger isn't thread-safe.  The logger class should be per instance.

If no logging isn't turned on, then no objects are created though.  So if people are really perf concerned, then don't use logging.